src/components/MessageBar/MessageBar.tsx

```diff
@@ -39,7 +39,7 @@ export default class MessageBar extends React.PureComponent<MessageBarProps, {}>
       <Block textSize={TextSize.SMALL} className={this.getClasses()}>
         <FixedGridRow gutterSize={GutterSize.XXLARGE}>
           <FixedGridColumn>{children}</FixedGridColumn>
-          {actions && <FixedGridColumn fixed={true}>{actions}</FixedGridColumn>}
+          {actions && <FixedGridColumn fixed={true}><span className='y-message-bar__nowrap'>{actions}</span></FixedGridColumn>}
```

Comment:
it might make more sense to handle this in FixedGridColumn. can you explain the problem this is solving?

Reply:
That line is introducing a couple of eslint warnings:
- ' should be "
- Exceeds maximum line length of 100

Reply:
@swese44 should we pair on this today to fix any remaining issues, and get it out the door? That way people working in YamJS can start killing old icons.